target/scripts/startup/setup-stack.sh

```diff
@@ -754,38 +754,44 @@ function _setup_docker_permit
       postconf -e "mynetworks ="
       ;;
 
-    "host" )
-      _notify 'inf' "Adding ${CONTAINER_NETWORK}/16 to my networks"
-      postconf -e "$(postconf | grep '^mynetworks =') ${CONTAINER_NETWORK}/16"
-      echo "${CONTAINER_NETWORK}/16" >> /etc/opendmarc/ignore.hosts
-      echo "${CONTAINER_NETWORK}/16" >> /etc/opendkim/TrustedHosts
-      ;;
-
-    "network" )
-      _notify 'inf' "Adding docker network in my networks"
-      postconf -e "$(postconf | grep '^mynetworks =') 172.16.0.0/12"
-      echo 172.16.0.0/12 >> /etc/opendmarc/ignore.hosts
-      echo 172.16.0.0/12 >> /etc/opendkim/TrustedHosts
-      ;;
-
     "connected-networks" )
       for NETWORK in "${CONTAINER_NETWORKS[@]}"
```

Comment:
Could the generation of `CONTAINER_NETWORKS` be "inlined" to this `case` statement or is it needed somewhere else?

Reply:
I think that's possible.

But the same goes for `$CONTAINER_IP` and `$CONTAINER_NETWORK`. The generation of these values are also handled outside the case statements. I think that was made for better readability of the case blocks 🤷

Reply:
It's not mandatory for this PR, just wanted to have an opinion. We can leave this as is.